src/modules/commander/commander.cpp

```diff
@@ -2781,6 +2786,7 @@ int commander_thread_main(int argc, char *argv[])
 		}
 
 		was_armed = armed.armed;
+		was_landed = land_detector.landed;
```

Comment:
-			was_landed = land_detector.landed;
I moved it to here, because "was_landed" in "else if (((!was_armed && armed.armed) || (was_landed && !land_detector.landed))" also is used.